### PR TITLE
Contact page

### PR DESCRIPTION
**Pull request by Znarfois**
- [x] Contact Section
- [x] Stay up-to-date-section
- [x] Clickable
- [x] Responsive

image
image

### CODE REVIEW

**Comment by Hjkun77** [comment]
@Znarfois so is this still in progress? Please make it responsive and clickable first

Also instead of `npm install` can you use `yarn install` so that consistent with @tanzellgo

**Comment by Hjkun77** [comment]
Why does your branch still have the old files? Did you git fetch origin before working on this branch?

File: src/pages/Contact.vue [hunk 1]
```diff
@@ -1,17 +1,195 @@
...
-    <h1>Contact Page</h1>
-  </main>
+    <main>
+        <div class="contact-section"> 
+            <h1>Contact</h1>
+            <div class="contact-people">
+                <div class="contact-person">
+                    <h2>Lance Lao</h2>
+                    <span>Events Lead</span>
+                    <p>christopher.lao@obf.ateneo.edu</p>
```
Comment (Hjkun77) [inline src/pages/Contact.vue]: change this to a `a href="mailto:christopher.lao@obf.ateneo.edu">christopher.lao@obf.ateneo.edu</a>`

File: src/pages/Contact.vue [hunk 2]
```diff
@@ -1,17 +1,195 @@
...
+                    <div class="socmed-content">
+                        <h2>LinkedIn</h2>
+                        <p>dsc-ateneo</p>
+                    </div>
+                </div>
+                <div class="socmed">
+                    <img src="../components/assets/ig_filled.png">
+                    <div class="socmed-content">
+                        <h2>Instagram</h2>
+                        <p>@dsc.ateneo</p>
```
Comment (Hjkun77) [inline src/pages/Contact.vue]: use ahref as well for linking

**Review by Hjkun77 — CHANGES_REQUESTED** [review]
Fix the code and remove the images from the previous version since we don't need them

**Comment by Hjkun77** [comment]
Remove 
`src/assets/Ellipse 46.png`
`src/assets/frabau.png`
`src/assets/shad.png`

**Review by tanzellgo — APPROVED** [review]
nice job bro just did some minor hot fix

**Pull request merged**